2014CPS-2009PUF/3AdjustFILSTV2014.sas

```diff
@@ -0,0 +1,55 @@
+*******************************************************************************************;
+***                                                                                     ***;
+***                                                                                     ***;
+***                          State Modeling Project                                     ***;
+***                                                                                     ***;
+***                                                                                     ***;
+*******************************************************************************************;
+OPTIONS PAGESIZE=59 LINESIZE=160 CENTER ; /* LANDSCAPE */
+*****
+	PRODUCTION FILE
+*****;
+LIBNAME EXTRACT "C:\Users\anderson.frailey\Documents\";
+*****
+	PROCESS THE PRODUCTION FILE
+
+	NOTE: WE USE A TEMPORARY DATA SET FIRST UNTIL
+	 	  THE SOI CONTROL TOTAL IS REACHED.
+	17-SEP-2015
+*****;
+DATA EXTRACT.CPSRETS2014(COMPRESS=YES);
+SET EXTRACT.CPSRETS2014;
+RETAIN ISEED1 21679 ISEED2 665533 ;
+*****
+	Adjust FILST Variable
+*****;
+IF( FILST EQ 0.) THEN

```

Comment:
@MattHJensen here is where the random number generator comes into play. I haven't found any articles saying the version of SAS would change the output, but I did find [this](http://blogs.sas.com/content/iml/2013/07/10/stop-using-ranuni.html) blog post from 2013 saying SAS stopped offering support for this particular function so it's reasonable to assume it hasn't been developed since at least 2013.